### DISCUSSION TITLE
Embed iframe

### DISCUSSION THREAD

**Discussion by baBaraga** [GetPublii/Publii#1442] [Help]
Im having trouble embedding a form using an iframe. Seems to be a random problem, it either partially shows or using the block builder it does not show at all. using the <html> tag.

**Comment by candidexmedia** [comment]
Try this: 

Paste the iframe’s code in the HTML Block (for the Block Editor) or the HTML source code page (for the WYSIWYG Editor). Make sure to add the `data-responsive="false"` attribute to the iframe.

Often times, the iframes are there but hidden because of the way Publii handles their sizing (see [Bob’s comment in another thread](https://forum.getpublii.com/topic/audio-player-embed/#post-8414) and the [doc page](https://getpublii.com/dev/how-the-post-content-is-modified-before-the-output/#make-iframes-responsive))

Video demo: https://i.imgur.com/7gPfbv7.mp4

[Source](https://forum.getpublii.com/topic/best-way-to-create-forms-for-publii-sites/#post-10613)

**Reply by baBaraga** [reply]
That worked!  thank you so much for the help!

**Reply by paulkater** [reply]
That saved me too. Thank you.

**Comment by Leetcore** [comment]
`data-responsive="false"` works. I think it should be added default if you enter an iframe as an embedded video via editor.

**Reply by paulkater** [reply]
I agree 100%
